postgresql-apb/roles/postgresql-apb-openshift/tasks/main.yml

```diff
@@ -87,5 +87,6 @@
   shell: 'echo "{\"POSTGRES_HOST\": \"{{ postgres_service.service.spec.cluster_ip }}\", \"POSTGRES_PORT\": \"5432\", \"POSTGRES_USER\": \"{{ postgresql_user }}\", \"POSTGRES_PASSWORD\": \"{{ postgresql_password }}\", \"POSTGRES_DB\": \"{{ postgresql_database }}\"}" | base64 -w 0'
   register: encoded_bind_credentials
 
-- debug:
-    msg: "<BIND_CREDENTIALS>{{ encoded_bind_credentials.stdout }}</BIND_CREDENTIALS>"
+- copy:
+   content="<BIND_CREDENTIALS>{{ encoded_bind_credentials.stdout }}</BIND_CREDENTIALS>"
```

Comment:
Now that bind credentials are in their own file instead of scraping a log, do we still need to use `<BIND_CREDENTIALS>` tags?  Can the broker just read it without having to look for and omit the tags?